.github/workflows/encrypted-messaging.yml

```diff
@@ -0,0 +1,32 @@
+name: encrypted-messaging
```

Comment:
We can run all tests by simply calling `make` command in the root folder, actually this was the first thing I did here: https://github.com/Lightprotocol/psp-examples/actions/workflows/makefile.yml

I separated them to different workflows because in this case we can run only one test when needed and skip all the others (each test takes few minutes, all together more than 30 mins)

Reply:
makes sense